tests/unit/containers/tutor-profile/ProfileInfo.spec.jsx

```diff
@@ -12,6 +12,52 @@ Object.assign(window.navigator, {
   }
 })
 
+const userData = {
+  _id: '64822a1433ebe4890079bb60',
+  role: ['tutor'],
+  firstName: 'Commander',
+  lastName: 'Shepard',
+  email: 'rubber883@gmail.com',
+  mainSubjects: {
+    student: [],
+    tutor: [
+      {
+        _id: '645b9f4a1c0272f5cde0e11e',
+        name: 'Danish',
+        category: '6459347d943e375d1c0a1912',
+        totalOffers: 0
+      },
+      {
+        _id: '6422d995d898aa732d038e8f',
+        name: 'Guitar',
+        category: '6421ed8ed991d46a84721dfa',
+        totalOffers: 4
+      },
+      {
+        _id: '6422ad6a74c1353b96c7c132',
+        name: 'Web design',
+        category: '6421ed8ed991d46a84721df4',
+        totalOffers: 9
+      }
+    ]
+  },
+  photo: '../src/John_Shepard_29.jpeg',
+  professionalSummary:
+    'Some amount of text regarding the proficiency.Some amount of text rega',
+  address: {
+    city: 'Lviv',
+    location: 'Ukraine'
+  },
+  averageRating: {
```

Comment:
averageRating and totalReviews always have both student and tutor properties

Reply:
Added